lib/queue.js

```diff
@@ -1,27 +1,30 @@
 class Queue {
```

Comment:
👀 While your implementation of the Queue does work, the instructions ask you to use a circular buffer in your implementation!